pkg/pillar/types/global.go

```diff
@@ -295,6 +295,10 @@ const (
 	SyslogLogLevel GlobalSettingKey = "debug.syslog.loglevel"
 	// KernelLogLevel global setting key
 	KernelLogLevel GlobalSettingKey = "debug.kernel.loglevel"
+	// SyslogRemoteLogLevel global setting key
+	SyslogRemoteLogLevel GlobalSettingKey = "debug.syslog.remote.loglevel"
+	// KernelRemoteLogLevel global setting key
+	KernelRemoteLogLevel GlobalSettingKey = "debug.kernel.remote.loglevel"
```

Comment:
Don't you also need to add something for the agent-specific remote loglevel? Or that is in a separate PR to add code to the zedagent parser?

Reply:
it's already there https://github.com/lf-edge/eve/blob/46c6ba43e35732aa5945f0a814214f545f9c1fa3/pkg/pillar/types/global.go#L350-L355